docs/content/developer/iota-101/objects/shared-object-example.mdx

```diff
@@ -0,0 +1,55 @@
+
+## Create Move Package
+
+To [create a new move package](../../getting-started/create-a-package.mdx) run the following command:
+
+```shell
+iota move new transfer-to-shared-object
+```
+## Package Overview
+
+In this example, we will create a package that allows users to create new shared objects of type [`SharedCoins`](https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L11-L14), delete them, add new coins to the [`SharedCoins`](https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L11-L14), and transfer coins to a recipient address from a [`SharedCoins`](https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L11-L14).  
+
+[`SharedCoins`](https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L11-L14) will hold IOTA coins, enabling anyone to deposit and withdraw coins from it.
+
+### [`create`](https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L16-L22)
+
+This function initializes a shared object with an empty array named `coins`.
+
+```move reference
+https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L16-L22
+```
+
+### [`delete`](https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L25-L29)
+
+This function destroys the `SharedCoins` shared object.
+
+```move reference
+https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L25-L29
+```
+
+### [`deposit_coin`](https://github.com/iotaledger/iota/blob/0494dad2b1d64c054795bac75342597e2eda1e9a/examples/move/transfer-to-shared-object/sources/shared_coins.move#L31-L38)
+
+This function deposits IOTA coins into `SharedCoins` by adding them to the `coins` array of `SharedCoins`.  
+Before using this function, ensure that some IOTA coins have been transferred to address of the `SharedCoins` object. For the `coin` parameter, provide the address of the IOTA coin you transferred to `SharedCoins`. For further details, refer to [Receiving](../../../references/framework/iota-framework/transfer.mdx#0x2_transfer_Receiving) and [public_receive](../../../references/framework/iota-framework/transfer.mdx#function-public_receive).
```

Comment:
Sorry I forgot `the`, also not sure how clear "address" is here, every object ID is an address, but might be not clear for everyone
```suggestion
Before using this function, ensure that some IOTA coins have been transferred to the address/object ID of the `SharedCoins` object.
For the `coin` parameter, provide the address/object ID of the IOTA coin you transferred to `SharedCoins`. For further details, refer to [Receiving](../../../references/framework/iota-framework/transfer.mdx#0x2_transfer_Receiving) and [public_receive](../../../references/framework/iota-framework/transfer.mdx#function-public_receive).
```